plugins/source/gcp/client/client.go

```diff
@@ -24,6 +24,9 @@ type Client struct {
 	ProjectId string
 	// Logger
 	logger zerolog.Logger
+
+	// This is used if we don't have a context in the call
+	ctx context.Context
```

Comment:
This is an unfortunate addition but we need some kind of context for the OrgMultiplexer, don't want to use context.BG

Reply:
Thx for pointing out. I'll take a look if there is a way to avoid it.

Reply:
I think in that case it should be called on `New` and then cached on the client level and the table should just get the cached data instead of reusing unrelated `ctx`

Reply:
@yevgenypats what do you mean? ctx is already set on `New` and it's copied from the source client to the multiplexed clients if the OrgMultiplexer is used.